NangPaGo-be/src/main/java/com/mars/NangPaGo/common/exception/NPGExceptionType.java

```diff
@@ -32,6 +32,7 @@ public enum NPGExceptionType {
     NOT_FOUND_RECIPE(HttpStatus.NOT_FOUND, "레시피를 찾을 수 없습니다."),
     NOT_FOUND_COMMENT(HttpStatus.NOT_FOUND, "댓글을 찾을 수 없습니다."),
     NOT_FOUND_INGREDIENT(HttpStatus.NOT_FOUND, "등록된 식재료 정보를 찾을 수 없습니다."),
+    NOT_FOUND_RECIPE_FAVORITE(HttpStatus.NOT_FOUND, "레시피 즐겨찾기 목록이 없습니다."),
```

Comment:
1. RECIPE 관련 예외끼리 묶여있으면 더 읽기 좋을 것 같습니다. 

```suggestion
    NOT_FOUND_RECIPE(HttpStatus.NOT_FOUND, "레시피를 찾을 수 없습니다."),
    NOT_FOUND_RECIPE_FAVORITE(HttpStatus.NOT_FOUND, "레시피 즐겨찾기 목록이 없습니다."),
    NOT_FOUND_COMMENT(HttpStatus.NOT_FOUND, "댓글을 찾을 수 없습니다."),
    NOT_FOUND_INGREDIENT(HttpStatus.NOT_FOUND, "등록된 식재료 정보를 찾을 수 없습니다."),
```

2. 문장이 조금 어색한 것 같아요.
추천: 즐겨찾기한 레시피가 없습니다.

Reply:
반영완료입니다